src/app/(homepage)/page.tsx

```diff
@@ -122,23 +122,25 @@ export default function Homepage() {
           title="Running on Filecoin Onchain Cloud"
           description="Projects using Filecoin Onchain Cloud to power verifiable, onchain applications and infrastructure."
         >
-          <CardGrid as="ul" variant="smTwoLgThreeWider">
-            {runningOnFilecoinOnchainCloud.map(
-              ({ title, description, image, link }) => (
-                <Card
-                  key={title}
-                  as="li"
-                  title={title}
-                  description={description}
-                  image={image}
-                  cta={{
-                    href: link,
-                    text: `Explore ${title}`,
-                  }}
-                />
-              ),
-            )}
-          </CardGrid>
+          <div className="[&>ul]:gap-y-20">
```

Comment:
@barbaraperic Is this a one off? Or a design that Filipa is pursuing for more grids? If it's the latter, we should want to add a variant at UI-Filecoin level.

If it's the former, then we might not want to use `CardGrid`..